components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java

```diff
@@ -199,8 +199,18 @@ private Exception createExceptionFromFault(Fault fault) {
                 throw new RuntimeCamelException(e);
             }
         }
+
+	Object detailObj = faultDetail.getAny().get(0);
+
+        if (!JAXBIntrospector.isElement(detailObj)) {
```

Comment:
Sorry - wrote this in a rush this morning and found that method. Will clean it up